en/08_Changelogs/5.3.0.md

```diff
@@ -51,6 +52,26 @@ Out of the box `silverstripe/assets` provides the new [`InterventionImageFileCon
 
 See [convert a file to a different format](/developer_guides/files/file_manipulation/#file-conversion) for more details.
 
+### Improve customisability of rendered images {#image-rendering}
+
+We've aligned how images added to an HTMLEditorField and SS template images are rendered. The markup up for both `<img>` tag is now generated from the same template file. This will make it easier to customise how images are rendered on your site.
```

Comment:
I assume when you say "SS template" you mean "when you invoke the `forTemplate()` method for an image" i.e. `$MyImage`.

Should probably be more explicit about that, especially in the `02_images.md` file. It's very common for people to write explicit `img` tags for each use of an image in a template, so the distinction is important.

Reply:
I adopted the "invoked in templates" language.